neo/SmartContract/InteropService.cs

```diff
@@ -537,9 +539,10 @@ private static bool Contract_Call(ApplicationEngine engine)
 
             StackItem method = engine.CurrentContext.EvaluationStack.Pop();
             StackItem args = engine.CurrentContext.EvaluationStack.Pop();
+            string methodStr = method.GetString();
             ContractManifest currentManifest = engine.Snapshot.Contracts.TryGet(engine.CurrentScriptHash)?.Manifest;
 
-            if (currentManifest != null && !currentManifest.CanCall(contract.Manifest, method.GetString()))
+            if (currentManifest != null && !currentManifest.CanCall(contract.Manifest, methodStr))
```

Comment:
@neo-project/core @erikzhang current Manifest could be null? i think that it is a MUST

```suggestion
            if (currentManifest == null || !currentManifest.CanCall(contract.Manifest, methodStr))
```

Reply:
Null makes no sense.. its mandatory now. Even native contracts should have one.

Reply:
Is not possible to remove the null check, because otherwise we can call from non stored contract to other one

Reply:
Checked the called contract, not the caller